### PR TITLE
Normalize the package name for Doc.Ms readme

### PR DESCRIPTION
**Pull request by sima-zhu**
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=713510&view=logs&j=df6dbc9d-9cb6-55ac-00c8-bc457d14297a&t=44cfd120-b037-5184-ffdd-7a7fc47afe90

The readme we get from the workflow: `sdk-template-readme.md` which is the same as before.

### CODE REVIEW

File: eng/scripts/Language-Settings.ps1 [hunk 1]
```diff
@@ -64,6 +64,7 @@ function Get-java-PackageInfoFromPackageFile ($pkg, $workingDirectory)
   [xml]$contentXML = Get-Content $pkg
 
   $pkgId = $contentXML.project.artifactId
+  $docsReadMeName = $pkgId -replace "azure-" , ""
```
Comment (weshaggard) [inline eng/scripts/Language-Settings.ps1:67]:
```suggestion
  $docsReadMeName = $pkgId -replace "^azure-" , ""
```

File: eng/common/pipelines/templates/steps/cosmos-emulator.yml [hunk 2]
```diff
@@ -9,92 +9,8 @@ steps:
       Write-Host "Target Dir: $targetDir"
       msiexec /a ${{ parameters.EmulatorMsiUrl }} TARGETDIR=$targetDir /qn | wait-process
     displayName: Download and Extract Public Cosmos DB Emulator
-  - powershell: |
```
Comment (weshaggard) [inline eng/common/pipelines/templates/steps/cosmos-emulator.yml:12]: Looks like you have some outdated changes.

Comment (sima-zhu) [reply eng/common/pipelines/templates/steps/cosmos-emulator.yml:12]: I am testing my changes with the tools change. Will revert the changes.

File: eng/common/scripts/Package-Properties.ps1 [hunk 3]
```diff
@@ -10,8 +10,6 @@ class PackageProps
     [string]$ReadMePath
     [string]$ChangeLogPath
     [string]$Group
-    [string]$SdkType
```
Comment (weshaggard) [inline eng/common/scripts/Package-Properties.ps1:13]: This is undoing other changes.

**Review by weshaggard — APPROVED** [review]

**Pull request merged**